### DISCUSSION TITLE
Should each plot get some mojos?

### DISCUSSION THREAD

**Discussion by pluto2021** [Chia-Network/chia-blockchain#5919] [Ideas]
To keep the netspace growing healthy, would it make sense for each valid plot to be rewarded with some mojos to cover the basic cost (statistical average) to generate the plot and keep the plot online? say if the plot got offline, the reward would start to decay with time, and after it is back online, it would start to accumulate till reaching the per plot reward amount?  This would also generate transactions initiated evenly on the network to help ensure equal chance of block reward?

**Comment by tiny0little** [comment]
pools do that. they share reward with you, proportionately to the total size of your plots, regardless if you mined that block or not
in case of solo, it is a lottery...
